utils.sh

```diff
@@ -487,4 +487,24 @@ function auth_template_and_removetmp(){
     removetmp
 }
 
+function configure_samples_operator() {
```

Comment:
About the name, this seems a generic configuration step. Some additional comments describing/motivating the two blocks codes below would be useful (looks like they could be split in two different funcs)

Reply:
Yes totally agree. After having a fully working PR, I'm going to elaborate parts to describe what are their purpose and possibly two different functions.

Reply:
nit: I don't see specifically the `configure_samples_operator` as generic util function, I think it could be defined as well in ` 06_create_cluster`